build/packages.targets

```diff
@@ -17,51 +18,59 @@
     <ItemGroup>
         <!-- PackageDownload -->
         <PackageDownload Update="Microsoft.Build" Version="[15.1.262-preview5]" />
-        <PackageDownload Update="Microsoft.VisualStudio.ProjectSystem.Managed" Version="[16.7.0-beta1-65318-02]" />
-        <PackageDownload Update="Microsoft.VisualStudio.ProjectSystem.Managed.VS" Version="[16.7.0-beta1-65318-02]" />
+        <PackageDownload Update="Microsoft.VisualStudio.ProjectSystem.Managed" Version="[$(ProjectSystemManagedVersion)]" />
+        <PackageDownload Update="Microsoft.VisualStudio.ProjectSystem.Managed.VS" Version="[$(ProjectSystemManagedVersion)]" />
         <PackageDownload Update="NuGet.Core" Version="[$(NuGetCoreV2Version)]" />
         <PackageDownload Update="VSLangProj140" Version="[14.0.25029]" />
         <PackageDownload Update="VSLangProj150" Version="[1.0.0]" />
         <!-- PackageReference -->
         <PackageReference Update="Lucene.Net" Version="3.0.3" />
         <PackageReference Update="MessagePack" Version="2.2.60" />
-        <PackageReference Update="Microsoft.Build" Version="$(MicrosoftBuildPackageVersion)" />
+        <PackageReference Update="Microsoft.Build" Version="$(MicrosoftBuildPackageVersion)" ExcludeAssets="runtime" />
```

Comment:
Can we move the exclude assets to project file instead?

Reply:
Removed it instead. It's no longer necessary.